frontend/src/metabase/dashboard/containers/DashboardHeader.unit.spec.tsx

```diff
@@ -0,0 +1,68 @@
+import fetchMock from "fetch-mock";
+import {
+  renderWithProviders,
+  screen,
+  waitForElementToBeRemoved,
+} from "__support__/ui";
+import {
+  createMockCollection,
+  createMockDashboard,
+} from "metabase-types/api/mocks";
+import DashboardHeader from "./DashboardHeader";
+
+const setup = () => {
+  fetchMock.get(
+    "path:/api/collection/10",
+    createMockCollection({
+      name: "Performance",
+      id: 10,
```

Comment:
I think this test relies on the collection entity loader shoving in the type prop. If we just rely on the dashboard collection hydration, we can just manually populate it in the `dashboard.collection` right?

Reply:
🤦‍♂️ You're right. I'll add the type directly in the mock